cmd/livenessprobe/main.go

```diff
@@ -42,14 +42,15 @@ var (
 )
 
 type healthProbe struct {
-	driverName string
+	driverName     string
+	metricsManager *metrics.CSIMetricsManager
```

Comment:
Does this need to be a pointer? The returned object is already a struct pointer but with an interface type

Reply:
Good point! Fixed!